.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "contrib/timpi"]
 	path = contrib/timpi
 	url = ../../libMesh/TIMPI
+[submodule "m4/autoconf-submodule"]
+	path = m4/autoconf-submodule
+	url = git@github.com:libMesh/autoconf-submodule
```

Comment:
We need to use the relative path here, for people that don't use ssh keys... the same as the TIMPI url above.

Reply:
Huh, I agree; I'm not sure why it didn't give me a relative path.  IIRC I didn't do anything with the submodule add command here that I hadn't done elsewhere.

Reply:
And I've got the same non-relative paths elsewhere, except for MetaPhysicL and TIMPI themselves, so I guess I was using a different setting back then?

Reply:
Yeah I think you have to go in and edit it by hand. I did that for the TIMPI one in 3746e900a3d4a906a1b656a50814096389d66aee for example.